VP-fitness-frontend/app/src/main/java/com/example/neil/vp_fitness_frontend/OnboardingFragmentActivity.java

```diff
@@ -0,0 +1,230 @@
+import android.support.v4.view.ViewPager;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+
+import com.avontell.fontutil.FontUtil;
+import com.avontell.pagerindicatorbinder.IndicatorBinder;
+import com.matthewtamlin.sliding_intro_screen_library.indicators.DotIndicator;
+
+import java.util.HashMap;
+
+public class OnboardingFragmentActivity extends AppCompatActivity {
+
+    /**
+     * The {@link android.support.v4.view.PagerAdapter} that will provide
+     * fragments for each of the sections. We use a
+     * {@link FragmentPagerAdapter} derivative, which will keep every
+     * loaded fragment in memory. If this becomes too memory intensive, it
+     * may be best to switch to a
+     * {@link android.support.v4.app.FragmentStatePagerAdapter}.
+     */
+    private SectionsPagerAdapter mSectionsPagerAdapter;
+
+    /**
+     * Context for this activity
+     */
+    private Context context;
+
+    /**
+     * Mapping for all onboarding screens
+     */
+    HashMap<String, Fragment> fragments = new HashMap<>();
+
+    /**
+     * The {@link ViewPager} that will host the section contents.
+     */
+    private ViewPager mViewPager;
+    protected int pageId;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.onboarding_fragment);
+
+        // Create the adapter that will return a fragment for each of the three
+        // primary sections of the activity.
+        mSectionsPagerAdapter = new SectionsPagerAdapter(getSupportFragmentManager());
+
+        // Set up the ViewPager with the sections adapter.
+        mViewPager = (ViewPager) findViewById(R.id.container);
+        mViewPager.setAdapter(mSectionsPagerAdapter);
+        mViewPager.setOffscreenPageLimit(10);
+        LinearLayout indicatorContainer = (LinearLayout) findViewById(R.id.indicator_cont);
+        int selectedImage = R.drawable.indicator_selected;
+        int unselectedImage = R.drawable.indicator_unselected;
+        final IndicatorBinder sample = new IndicatorBinder().bind(this,
+                mViewPager,
+                indicatorContainer,
+                selectedImage,
+                unselectedImage);
+        sample.setProgressStyle(false);
+        mViewPager.addOnPageChangeListener(new ViewPager.OnPageChangeListener() {
+            @Override
+            public void onPageScrolled(int position, float positionOffset, int positionOffsetPixels) {
+
+            }
+
+            @Override
+            public void onPageSelected(int position) {
+            }
+
+            @Override
+            public void onPageScrollStateChanged(int state) {
+
+            }
+        });
+    }
+
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
```

Comment:
There is no options menu for this activity, so delete this and the following `onOptionsItemSelected` method